### ISSUE TITLE
Duplicate labels on some fields 

### ISSUE THREAD

**Issue by andrebtlima** [CityOfBoston/Iterators#172]
Screengrab/videos (insert file names here)
2020 09 01, Form Label Errors.mp4

Link
https://registry-certs.digital-staging.boston.gov/marriageintention?step=partnerFormA

Node # (if known)

Type of User: Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 85

Trusted Tester Test
Forms

Brief Description
Several fields on the page have duplicative fields 

Steps to reproduce

1. Go to the Marriage license link and get to the second step in the process 
2. Check out the Output generated by ANDI for the form labels
3. City/Town field, and the fields in “Name of Mother / Parent 1” and “Name of Father / Parent 2” (First Middle Last and Maiden Name / Surname At Birth Or Adoption) shows a duplicate ID
a) Bug: field ID’s should be unique in order for the accessibility text to be processed correctly

Desired Result
• The form labels should all be unique

Actual Results
• All of the field IDs are unique in order for accessibility text to be process correctly.
• The fields with non-unique IDs are:
City/Town
First Middle Last
Maiden Name / Surname At Birth Or Adoption

**Comment by andrebtlima** [comment]
@phillipbentonkelly essentially what's going on here is that several fields have the same ID as another field. So the City/Town field for Birthplace has the same label as the City/Town fied for Residence, meaning that someone using a screen reader would have trouble telling the difference between the two fields. So the labels could be differentiated by appending Birthplace and Residence according to which field it is. Similar duplication happening for First Middle Last and  Maiden Name / Surname At Birth Or Adoption for Parent 1 and Parent 2

Heres a video that shows the issue: 

https://drive.google.com/file/d/1Aed4YiaJ5-1gNPMoHLatvfUYnzG3dIg4/view

**Comment by andrebtlima** [comment]
@david-iterators @jill-iterators this is ready to retest.

**Comment by jill-iterators** [comment]
@andrebtlima This was tested today and passed
